src/node_credentials.cc

```diff
@@ -123,7 +123,6 @@ bool SafeGetenv(const char* key,
   }
 
 fail:
-  text->clear();
```

Comment:
is this deliberate? (it doesn't seem relevant)

caveat: i haven't written C in 15–20 years

Reply:
Yes, if the `SafeGetEnv`h has a previous value, this overrides/removes it. This function already returns a boolean to distinguish a success/failure, so no need to clear the text as well.